reqMgrClient.py

```diff
@@ -399,8 +427,8 @@ def getWorkflowInfo(url, workflow):
     """
```

Comment:
Thanks Julian,
Actually I meant all the test you are running, not just this case. Do you have a script for that? Otherwise, I will just create one.